Packs/GoogleDocs/Integrations/GoogleDocs/GoogleDocs.py

```
@@ -357,15 +357,19 @@ def main():
 
     try:
         service = get_client(service_account_credentials, SCOPES, proxy, disable_ssl)
-        if demisto.command() == 'google-docs-update-document':
-            document, human_readable_text = batch_update_document_command(service)
-        elif demisto.command() == 'google-docs-create-document':
-            document, human_readable_text = create_document_command(service)
-        elif demisto.command() == 'google-docs-get-document':
-            document, human_readable_text = get_document_command(service)
-        else:
-            return_error("Command {} does not exist".format(demisto.command()))
-            return
+        match command:
+            case 'google-docs-update-document':
+                document, human_readable_text = batch_update_document_command(service, args)
+
+            case 'google-docs-create-document':
+                document, human_readable_text = create_document_command(service, args)
+
+            case 'google-docs-get-document':
+                document, human_readable_text = get_document_command(service, args)
+
+            case _:
+                return_error("Command {} does not exist".format(command))
+                return
```

let's raise instead of `return_error`, as it's caught in the _main_ part below